fix: Add form component page (#137)
lenra-io/components-api: docs/src/pages/components/form.mdx

import PropertyTable from '../../components/propertyTable';
import flexSchema from '../../../../api/components/form.schema.json';

> The Form component is a container that dispatch an event with all it containing fields data.

Reviewer: ```suggestion
The Form component is a container that dispatches an event with all its containing field values.
```
